invenio_app_ils/records_relations/views.py

```diff
@@ -77,37 +78,44 @@ def _get_record(self, record, pid, pid_type):
     def _validate_parent_child_creation_payload(self, payload):
         """Validate the payload when creating a new parent-child relation."""
         try:
-            parent_pid = payload.pop("parent_pid")
+            parent_pid_value = payload.pop("parent_pid_value")
             parent_pid_type = payload.pop("parent_pid_type")
-            child_pid = payload.pop("child_pid")
+            child_pid_value = payload.pop("child_pid_value")
             child_pid_type = payload.pop("child_pid_type")
         except KeyError as key:
             raise RecordRelationsError(
                 "The `{}` is a required field".format(key)
             )
 
-        return parent_pid, parent_pid_type, child_pid, child_pid_type, payload
+        return (
+            parent_pid_value,
+            parent_pid_type,
+            child_pid_value,
+            child_pid_type,
+            payload,
+        )
 
     def _create_parent_child_relation(self, record, relation_type, payload):
         """Create a Parent-Child relation.
 
         Expected payload:
 
             {
-                parent_pid: <pid_value>,
+                parent_pid_value: <pid_value>,
```

Comment:
nit: in other places we started to pass pid as object
```
{
type: 'abc`
value: 'ergg-134s3'
}
```
just a general reminder keep it consistent (I know that it was this way before)

Reply:
I know and I thought a lot about it, then I went for this because it is actually much more usable in code instead of an object and we have already a lot of objects in `relations`, WDYT?

I would honestly go for quite flat objects in general.

Reply:
good for me!